pkg/configuration/bb_worker/configuration.go

```diff
@@ -28,11 +28,20 @@ func setDefaultWorkerValues(workerConfiguration *pb.WorkerConfiguration) {
 	if workerConfiguration.CacheDirectoryPath == "" {
 		workerConfiguration.CacheDirectoryPath = "/worker/cache"
 	}
+	if workerConfiguration.MaximumCacheFileCount == 0 {
+		workerConfiguration.MaximumCacheFileCount = 10000
+	}
+	if workerConfiguration.MaximumCacheSizeBytes == 0 {
+		workerConfiguration.MaximumCacheSizeBytes = 1024 ^ 3
```

Comment:
Careful here, `1024 ^ 3` will not do what you expect it to do!

Reply:
Such a mistake! Don't know why. Thank's for spotting.